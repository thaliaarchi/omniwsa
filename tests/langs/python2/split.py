import platform
print "Python %s" % platform.python_version()

def expect_except(func):
    try: func()
    except Exception: pass
    else: raise Exception("expected exception")

expect_except(lambda: chr(-1))
expect_except(lambda: chr(256))
print [chr(c) for c in range(0, 256) if len(chr(c).split()) == 0]
