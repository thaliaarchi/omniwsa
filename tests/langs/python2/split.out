Python 2.7.18
['\t', '\n', '\x0b', '\x0c', '\r', ' ']
